.github/workflows/pypi_publish.yml

```diff
@@ -31,12 +31,11 @@ jobs:
       shell: bash -l {0}
       run: |
         mamba config --set always_yes yes --set changeps1 no
-        mamba install -q -n pypi fftw \
-                                 liblapack \
-                                 openblas \
+        mamba install -q -n pypi blas-devel \
+                                 fftw \
+                                 meson-python \
```

Comment:
should we require meson-python to be >= 0.14.0 ? As noted below, the convention of using the file name `meson.options` instead of `meson_options.txt` is for meson 1.1.+ and meson-python 0.14.+